packages/mdc-top-app-bar/README.md

```diff
@@ -0,0 +1,122 @@
+<!--docs:
+title: "Toolbars"
+layout: detail
+section: components
+excerpt: "A container for multiple rows of items such as application title, navigation menu, or tabs."
+iconId: toolbar
+path: /catalog/topappbar/
+-->
+
+# Top App Bars
```

Comment:
I realize we're inconsistent across packages right now (and Toolbar currently pluralizes this as well) but I'd vote for singular here, since you're likely only going to use one Top App Bar on a given page.